data-prepper-plugins/otel-trace-group-processor/README.md

```diff
@@ -46,6 +46,10 @@ Default is null.
 
 - `aws_region`: A String represents the region of Amazon OpenSearch Service domain, e.g. us-west-2. Only applies to Amazon OpenSearch Service. Defaults to `us-east-1`.
 
+- `aws_sts_role_arn`: A IAM role arn which the sink plugin will assume to sign request to Amazon OpenSearch Service. If not provided the plugin will use the default credentials.
```

Comment:
I don't see any changes made in `otel-trace-group-processor` except readme file, are these changes made in a different PR?

Reply:
Good note! It turns out that this processor uses the configuration from the `opensearch` sink project.

You can see the Gradle dependency here:

https://github.com/opensearch-project/data-prepper/blob/07742c4234732042336768e9432651af47c2c19c/data-prepper-plugins/otel-trace-group-processor/build.gradle#L16